java/src/jmri/jmrit/display/layoutEditor/LayoutTrack.java

```diff
@@ -80,7 +80,7 @@ public LayoutTrack(@Nonnull String ident, @Nonnull Point2D c, @Nonnull LayoutEdi
         this.ident = ident;
         this.center = c;
         this.layoutEditor = layoutEditor;
-        defaultTrackColor = ColorUtil.stringToColor(layoutEditor.getDefaultTrackColor());
+        defaultTrackColor = layoutEditor.getDefaultTrackColorColor();
```

Comment:
Why does this change trigger a static analysis warning when the prior line did not?
```
[ERROR] Write to static field jmri.jmrit.display.layoutEditor.LayoutTrack.defaultTrackColor from instance method new jmri.jmrit.display.layoutEditor.LayoutTrack(String, Point2D, LayoutEditor) [jmri.jmrit.display.layoutEditor.LayoutTrack] At LayoutTrack.java:[line 83] ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD
```

Reply:
SpotBugs isn't perfect.  From a quick look at the rule, I _think_ that it omitted the message on the prior form because it decided it was a write of the result of a static method call.  Apparently the rule author thought that setting a static member from a static method was out of scope for this rule.

Reply:
@bobjacobsen Thank you. That's what I thought, I wasn't sure. It did reveal a subtle bug: The default track color could be set per layout panel, but was really set for track items globally across the app, ignoring the focused layout panel's default track color. Does fixing that warrant a comment in the release notes?